docs(examples): Add Tailwindcss (#1373)
remix-run/remix: examples/tailwindcss/README.md

# Tailwindcss Example

Integrate Remix with tailwindcss.

## Preview

Open this example on [CodeSandbox](https://codesandbox.io/s/remix-tailwind-2x8pg):

> [![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/tailwindcss)

Reviewer: ```suggestion
[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/tailwindcss)
```

## Example

This example shows how to use Tailwind CSS (v3.0) with Remix. It follows the steps outlined in the official [Remix Styling docs](https://remix.run/docs/en/v1/guides/styling#tailwind).

> Check [app/root.tsx](./app/root.tsx) where tailwind is imported.

Reviewer 2: ```suggestion
Relevant files:
- [package.json](./package.json) where the tailwind CLI is used.
- [tailwind.config.js](./tailwind.config.js) where tailwind is configured.
- [app/root.tsx](./app/root.tsx) where tailwind is imported.
```

## Related Links

> [tailwindcss](https:/tailwindcss.com)

Reviewer: ```suggestion
[Tailwind CSS](https:/tailwindcss.com)
```

> [Tailwind CSS](https:/tailwindcss.com)

Reviewer 2: ```suggestion
[Tailwind CSS](https://tailwindcss.com)
```
